prow/plugins/approve/approve.go

```diff
@@ -243,9 +243,10 @@ func handleReview(log *logrus.Entry, ghc githubClient, oc ownersClient, config *
 		&state{
 			org:       re.Repo.Owner.Login,
 			repo:      re.Repo.Name,
+			branch:    pr.Base.Ref,
 			number:    re.PullRequest.Number,
-			body:      re.Review.Body,
-			author:    re.Review.User.Login,
+			body:      re.PullRequest.Body,
```

Comment:
This just made me realize that we shouldn't be calling `GetPullRequest` above! We can just use `re.PullRequest` where ever `pr` is used in this function and avoid spending an API token.

Reply:
Thanks for noticing that! I noticed it myself but decided to leave it to keep the diff small :)